### PR TITLE
Wallet Connect test cases 

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Added test cases for wallet connect webmode

### CODE REVIEW

File: packages/web/test/src/lib/walletconnect-mode.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    this.beforeEach(function () {
+        sender = algosdk.generateAccount();
+        receiver = algosdk.generateAccount();
+        connector = new WallectConnectSession(walletURL, new WalletConnect({
+            bridge: "https://bridge.walletconnect.org",
+            qrcodeModal: QRCodeModal,
+        }));
+    });
+
+    it("Should executeTx without throwing an error", function () {
```
Comment (rmeena840) [inline packages/web/test/src/lib/walletconnect-mode.ts:31]:
```suggestion
    it("Should executeTx not throw an error", function () {
```

File: packages/web/test/src/lib/walletconnect-mode.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        server: testnetURL,
+        port: "",
+    }
+    algodClient = algoexplorerAlgod(walletURL);
+
+    this.beforeEach(function () {
+        sender = algosdk.generateAccount();
+        receiver = algosdk.generateAccount();
+        connector = new WallectConnectSession(walletURL, new WalletConnect({
+            bridge: "https://bridge.walletconnect.org",
```
Comment (rmeena840) [inline packages/web/test/src/lib/walletconnect-mode.ts:26]: If it's already declared as constant then please use it else create one.

File: packages/web/test/src/lib/walletconnect-mode.ts [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                type: types.TransactionType.TransferAlgo,
+                sign: types.SignType.SecretKey,
+                fromAccount: sender,
+                toAccountAddr: receiver.addr,
+                amountMicroAlgos: 1e6,
+                payFlags: {},
+            };
+
+            const txnParams = await getSuggestedParams(algodClient)
+            const transactions: Transaction[] = connector.makeTx([execParams], txnParams);
```
Comment (rmeena840) [inline packages/web/test/src/lib/walletconnect-mode.ts:74]: Let's also check receiver's fund.

Comment (rmeena840) [reply packages/web/test/src/lib/walletconnect-mode.ts:74]: and sender's fund too

File: packages/web/test/src/lib/walletconnect-mode.ts [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+                sign: types.SignType.SecretKey,
+                fromAccount: sender,
+                toAccountAddr: receiver.addr,
+                amountMicroAlgos: 1e6,
+                payFlags: {},
+            };
+            const txnParams = await getSuggestedParams(algodClient)
+            assert.doesNotThrow(async () => {
+                await connector.makeAndSignTx([execParams], txnParams);
+            });
```
Comment (rmeena840) [inline packages/web/test/src/lib/walletconnect-mode.ts:92]: same here

File: packages/web/test/src/lib/walletconnect-mode.ts [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                type: types.TransactionType.TransferAlgo,
+                sign: types.SignType.SecretKey,
+                fromAccount: sender,
+                toAccountAddr: receiver.addr,
+                amountMicroAlgos: 1e6,
+                payFlags: {},
+            };
+            const txnParams = await getSuggestedParams(algodClient)
+            const signedTx = await connector.makeAndSignTx([execParams], txnParams);
+            assert.doesNotThrow(async () => {
```
Comment (rmeena840) [inline packages/web/test/src/lib/walletconnect-mode.ts:106]: same here

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed.

File: packages/web/test/src/lib/walletconnect-mode.ts (hunk 1 above)
Comment (Megha-Dev-19) [reply packages/web/test/src/lib/walletconnect-mode.ts:31]: I think without throwing makes more sense

File: packages/web/test/src/lib/walletconnect-mode.ts (hunk 2 above)
Comment (Megha-Dev-19) [reply packages/web/test/src/lib/walletconnect-mode.ts:26]: it is not declared as constant, and it shouldn't be declared one as we create the instance in `beforeEach`

File: packages/web/test/src/lib/walletconnect-mode.ts (hunk 1 above)
Comment (rmeena840) [reply packages/web/test/src/lib/walletconnect-mode.ts:31]: Think about it once before merge.

**Review by ratik21 — APPROVED** [review]
lgtm

File: packages/web/test/src/lib/walletconnect-mode.ts (hunk 1 above)
Comment (Megha-Dev-19) [reply packages/web/test/src/lib/walletconnect-mode.ts]: updated it with `Should run executeTx function without throwing an error`

**Pull request merged**